File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    answer = list(s)
    idx = 0
    for i in range(len(s)):
        if s[i] == ' ':
            idx = 0
            continue
        if idx % 2 == 0:
            answer[i] = answer[i].upper()
        else:
            answer[i] = answer[i].lower()
        idx += 1
    return ''.join(answer)

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.java
class Solution {
    public String solution(String s) {
        char[] c = s.toCharArray();
        int idx = 0;
        for (int i = 0; i < s.length(); i++) {
            if (c[i] == ' ') {
                idx = 0;
                continue;
            }
            if (idx % 2 == 0) {
                c[i] = Character.toUpperCase(c[i]);
            } else {
                c[i] = Character.toLowerCase(c[i]);
            }
            idx += 1;
        }
        
        String answer = String.valueOf(c);
        return answer;
    }
}